app/Test.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Test extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'subject_id', 'group_id', 'start', 'end',
```

Comment:
Please also add a **title** and a **description**

Reply:
The title should be a _string_ with 255 characters max length.
The description should have a _text_ type.